src/stories/StyledTabContentWrapper.tsx

```diff
@@ -0,0 +1,25 @@
+import { PropsWithChildren } from 'react';
+import { IconVariant } from './DfxIcon';
+import StyledIconButton from './StyledIconButton';
+
+export interface StyledTabContentWrapperProps extends PropsWithChildren {
+  showBackArrow?: boolean;
+  onBackClick: () => void;
```

Comment:
`onBackClick` should not be required if back arrow is not required.

Reply:
you're right, thanks.